modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/integration/TableDdlIntegrationTest.java

```diff
@@ -275,6 +275,97 @@ public void createTableOnExistingCache() {
         assertEquals(2, cache.size(CachePeekMode.PRIMARY));
     }
 
+    /**
+     * Creates table as select.
+     */
+    @Test
+    public void createTableAsSelectSimpleCase() {
+        executeSql("create table my_table as select 1 as i, 'test' as s");
+        List<List<?>> res = executeSql("select i, s from my_table");
+
+        assertEquals(1, res.size());
+
+        List<?> row = res.get(0);
+
+        assertEquals(1, row.get(0));
+        assertEquals("test", row.get(1));
+    }
+
+    /**
+     * Creates table with specified columns as select.
+     */
+    @Test
+    public void createTableAsSelectWithColumns() {
```

Comment:
let's add a few test to verify column names of just created tables

Reply:
Check for column names added to this test